applications/autonomous_worlds_on_substrate.md

```diff
@@ -0,0 +1,352 @@
+
+**0. Setup**
+Prior to gameplay, a set of 'recipes' are encrypted (see the scheme below), circuits are compiled and verification keys are created (via powers of tau) for each one locally, and then everything is encoded on-chain (potentially relying on some offchain storage solution). These represent the conditions that players must meet in order to unlock recipes in the game. In the initial version, there is no option for players to create their own recipes.
+
+**1. Exploration and Foraging**
+Players find ingredients through Twitter clues leading to URLs. For example:
+- Mandrake Root: Solve a riddle at a provided URL.
+- Dragon Scale: Complete a quest linked in a tweet.
+- Phoenix Feather: Solve a puzzle from a tweet link.
+
+Players use the knowledge learned here to 'claim' ingredients later on when attempting to unlock a recipe. 
+
+**2. Arcane Assembler (Crafting System)**
+Players use collected "ingredients" and "recipes" to craft spells. To craft a spell, a player must first unlock the recipe. Recipes are encrypted using our practical witness encryption scheme. Using this, players can only unlock recipes if they can prove they have solved some set of challenges. 
+
+Crafting Process:
+i. Select spell recipe and gather ingredients (knowledge learned in phase (1)).
+ii. Prepare a valid proof and provide it to the network 
+iii. The network authorities produce a decryption key for the caller
+iv. The caller uses the key to decrypt the secret message
+v. Produce a valid signature on the secret message and provide it to the network to mint a spell NFT (using onchain randomness).
+
+Example Spells:
+- Firestorm Spell: Requires Dragon Scale, Phoenix Feather, Sunstone.
+- Healing Waters: Requires Mermaid Tear, Moon Blossom, Crystal Shard.
+
+**3. Spell Usage and Progression**
+Spell Usage: Spells are minted and sold as NFTs.
+
+Example NFT Sales:
+
+Craft Firestorm Spell (mint as NFT) => list for sale.
+
+Progression: Earning in-game currency or rewards to:
+- Buy better foraging tools.
+- Unlock new areas.
+- Invest in crafting skills.
+- Crafting Failure
+- Crafting can fail. Failed attempts result in lost ingredients, requiring players to forage again for new ones.
+
+### Are you intending to raise more funds?
+There are no plans to raise additional funds right now, but we are open to discussing this in the future. The community reception of the game will be a significant factor in determining whether we will seek additional funding. Successful completion of proof-of-concept for the game mechanic will likely lead to further development. Crowd-funding is the most suitable funding model for the future.
+
+## 6. Detailed Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):**  3
+- **Total Costs:** 9900 DOT ~ $75,000  USD
+
+### Milestone 1: Light Client + Relayer
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  3
+- **Costs:** 3300 DOT (6.875 DOT/hr)

```

Comment:
Unless DOT Moons I think we should be good with 8 DOT/hr

Reply:
done, brings us up to 11,520 DOT total